app/src/main/java/com/example/puzzlepic/service/PictureService.kt

```diff
@@ -23,7 +23,7 @@ class PictureService {
      * Uses a callback to retrieve json data from network and handles response accordingly
      * @return an LiveArrayList of pictures.
      */
-    fun fetchPicture() : MutableLiveData<ArrayList<Picture>> {
+   internal fun fetchPicture() : MutableLiveData<ArrayList<Picture>> {
```

Comment:
Adding a visability modifier. Reference: https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces